src/vectorchord/usage/search.md

```diff
@@ -0,0 +1,72 @@
+# Search
+
+Get the nearest 5 neighbors to a vector
+
+```sql
+SELECT * FROM items ORDER BY embedding <-> '[3,1,2]' LIMIT 5;
+```
+
+## Operators
+
+These operators are used for distance metrics:
+
+| Name  | Description                |
+| ----  | -------------------------- |
+| <->   | squared Euclidean distance |
+| <#>   | negative dot product       |
+| <=>   | cosine distance            |
```

Comment:
```suggestion
| Name  | Description                |
| ----- | -------------------------- |
| `<->` | squared Euclidean distance |
| `<#>` | negative dot product       |
| `<=>` | cosine distance            |
```

On my machine, the characters here incorrectly have ligatures enabled. Code blocks must be used here.